aten/src/ATen/native/TensorShape.cpp

```diff
@@ -779,7 +779,11 @@ std::vector<Tensor> split_with_sizes(const Tensor& self, IntArrayRef split_sizes
 
 static inline std::vector<Tensor> get_stack_inputs(TensorList tensors, int64_t dim) {
   std::vector<Tensor> inputs(tensors.size());
+  at::IntArrayRef entry_shape = tensors[0].sizes();
   for (size_t i = 0; i < tensors.size(); ++i) {
```

Comment:
You could avoid a redundant test when `i == 0`

Reply:
Sure. I opted to make the fewest changes, but this is only one more line.

Reply:
Note: the get_stack_inputs also creates the stack, so changing `i = 0` to `i = 1` means completely skipping the first input. I added this back in.